### ISSUE TITLE
Future Idea: In-situ cement for housing

### ISSUE THREAD

**Issue by mcortez** [BobPalmer/MKS#736]
Use substrate and water (or liquid sulfur like the current Mars soil experiments) to make Duna/Mun'a/etc cement that can be used to construct more permanent structure's in-situ.

**Comment by jd284** [comment]
You mean like pouring a flat, level foundation on which to attach parts for the base?
Unless it is somehow able to directly remodel the actual planet surface (is that even editable?) rather than adding a procedural part on top, with the dodgy ground physics that just screams "kraken bait".

**Comment by mcortez** [comment]
No, I was thinking of something much more abstract.  A processing unit that generates a cement like resource (store it in tanks) and then a module that would spit out a fully completed part, in place ready to use -- much like EPL when using say the mallet and stakes.  Could use an "extruder" part, that could be mounted on a rover or other lander that spits out a hab part once there is enough cement resource.  The extruded part would be adjacent to the extruder's output nozzle.

Would be similar to the inflatable modules, but would be attached to the ground in whatever manner KIS/KAS attaches things to the ground and would have more attachment nodes.  Obviously you wouldn't be able to deflate them, but they could provide higher radiation protection for when that becomes a thing.

You would be able to do some of the missions that NASA and others are theorizing about now, where you send an automated processing facility ahead of a manned mission that would process local resources and begin building a base of operations.

**Comment by BobPalmer** [comment]
I have some ideas....
